doc/source/deployment/verify.rst

```diff
@@ -17,3 +17,143 @@ services are functioning as expected by running the following commands:
    export OS_CLOUD='openstack'
    openstack endpoint list
    openstack server list
+
+OpenStack Tempest Testing
+=========================
+
+After the deployment of SUSE Containerized OpenStack has completed, it is possible to run
+OpenStack Tempest tests against the core services in the deployment using the `run.sh` script.
+Before running Tempest tests, it will be necessary to manually configure OpenStack network
+resources and provide a few configuration parameters in the ${WORKDIR}/env/extravars file.
+
+Setting Up An External Network And Subnet in OpenStack
+------------------------------------------------------
+
+To set up an external network and subnet in OpenStack, the following commands can be run from a
+shell on the `Deployer` node.
+
+.. code-block:: console
+
+   export OS_CLOUD=openstack
+   openstack network create --provider-network-type flat --provider-physical-network public \ 
+     --external ext-net
```

Comment:
Instead of ext-net, match the example command with public which is the default value.
openstack_external_network_name: "public"